CommunityFridgeMapApi/functions/fridges/v1/app.py

```diff
@@ -0,0 +1,54 @@
+import os
+import logging
+import json
+
+try:
+    from db import get_ddb_connection, Fridge
+except:
+    # If it gets here it's because we are performing a unit test. It's a common error when using lambda layers
+    # Here is an example of someone having a similar issue:
+    # https://stackoverflow.com/questions/69592094/pytest-failing-in-aws-sam-project-due-to-modulenotfounderror
+    from dependencies.python.db import get_ddb_connection, Fridge
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class FridgeHandler:
+    @staticmethod
+    def get_fridge_id(event: dict):
+        """
+        Gets the fridge_id from the event dictionary
+        """
+        fridge_id = None
```

Comment:
nit: entire block can be written as
`return event.get("pathParameters", {}).get("fridge_id", None)`

Reply:
event["pathParameters] is always present and can be `None`.

So event.get("pathParameters", {}) returns None when a fridge_id is not passed in and causes an error. 

I originally assumed that if it was None that it would default to {}, but that's not the case

Reply:
oh that's neat! You're good at pythonic coding haha

This would sacrifice readability over conciseness 🤔. Well I suppose readability is subjective 😅

I'm good with either. Does anyone else have a preference @0916dhkim @sbeesmiley 

For some reason I also like: 

```
pathParameters = event.get("pathParameters", {}) or {}
fridge_id = pathParameters.get("fridge_id", None)
```

Reply:
If we are sure `event["pathParameters"]` is always present, I would do this:
```
path_parameters = event["pathParameters"] or {}
fridge_id = path_parameters.get("fridge_id")
```
It's a shame that Python does not have optional chaining syntax.

Reply:
done 646cfa3c0f570cb314a32521eedc099b8d321563